### DISCUSSION TITLE
Idea: create client for xelaj/mtproto

### DISCUSSION THREAD

**Discussion by quenbyako** [charmbracelet/inspo#39] [Ideas]
**Is your feature request related to a problem? Please describe.**
I want to create CLI client for telegram messenger, based on [this library](https://github.com/xelaj/mtproto). Currently i just don't have enough time to implement it, but i actively maintaining this package.

**Describe the solution you'd like**
I want to see basic chat client with additional features like scripting, automatic replies, etc, main target is to make client as most scriptable as it can be. And pretty looking, of course

**Include projects that are similar in nature**
There is c-based alternative, [tg](https://github.com/vysheng/tg), but i just want to make client more colorful and usable

**Comment by bashbunni** [comment]
Hey I'm just converting all of the project ideas in this repo to discussions instead of issues :)
